src/js/tests/unit/ckeditor-loader.js

```diff
@@ -1,92 +1,95 @@
 $(function () {
   'use strict';
 
-  QUnit.module('ckeditor-loader', {
-    setup: function () {
+  describe('CKEditor loader plugin', function () {
+
+    beforeEach(function () {
       jQuery.fn.ckeditor = function () {};
-    },
+    });
 
-    teardown: function () {
+    afterEach(function () {
       delete jQuery.fn.ckeditor;
       $('html').attr('lang', null);
-    },
-  });
-
-  // Only Data API tests are needed as the rest of the functionality is provided by the ckeditor
-  // jQuery adapter
-
-  // Data-api tests
-  // ======================
-  QUnit.test('should init ckeditor on page load with no attribute', function () {
```

Comment:
I think we do not want to loose this test.

Reply:
All tests have been rewritten and also this test can be found in the rewritten form.